csharp/ql/test/query-tests/Security Features/CWE-201/ExposureInTransmittedData/ExposureInTransmittedData.expected

```diff
@@ -1,21 +1,13 @@
 edges
+| ExposureInTransmittedData.cs:16:32:16:39 | access to local variable password | ExposureInTransmittedData.cs:16:32:16:39 | access to local variable password |
+| ExposureInTransmittedData.cs:20:32:20:44 | call to method ToString | ExposureInTransmittedData.cs:20:32:20:44 | call to method ToString |
+| ExposureInTransmittedData.cs:24:32:24:41 | access to property Message | ExposureInTransmittedData.cs:24:32:24:41 | access to property Message |
```

Comment:
This is because the shared implementation represents paths of length 0 as paths of length 1 by adding a synthetic sink node. This is also the reason why the `nodes` predicate is no longer needed. (ping @aschackmull)

Reply:
(Note that these are therefore only additions to the `edges` query predicate, not the `select` predicate.)

Reply:
True. Although we might need to add a `nodes` predicate in order to save the `PathNode.toString` as labels, but that's a separate concern.